src/System.Net.Mail/src/System/Net/Mail/SmtpException.cs

```diff
@@ -109,12 +110,13 @@ internal SmtpException(string message, string serverResponse) : base(message + "
 
         void ISerializable.GetObjectData(SerializationInfo serializationInfo, StreamingContext streamingContext)
         {
-            base.GetObjectData(serializationInfo, streamingContext);
+            GetObjectData(serializationInfo, streamingContext);
```

Comment:
missed `_statusCode = (SmtpStatusCode)serializationInfo.GetInt32("Status");`

Reply:
should tests have caught that?

Reply:
I focused on adding general exception tests that guarantee that all the base exception data gets serialized correctly for all tests. For some cases I had to fine tune the tests but I didn't add tests for additional serialized data. This could be done in future.

Reply:
Thanks, I will add it.